Organizing files in a separate directory

gustafsilva opened the pull request:
Organizing files in a separate directory and updating location in container `html-tools`. #312 #313

gustafsilva commented on docker-compose.yml:
```diff
@@ -15,7 +15,7 @@ services:
     links:
       - apy
     volumes:
-      - .:/root/apertium-html-tools
```
I saw now that it has failed the `html-tools` container. :grimacing:
Changing this line, does anything else need to be done?
I tested it as it as before and website ran perfectly... :+1:

gustafsilva commented:
I do not know what GCI is, can you explain? :relaxed: I'm now starting to contribute with open source, i'm still learning how this world works. :blush:
I changed the routes in the `Makefile`, can i run the tests without giving a pull request to see i there were any errors? :sweat_smile:

sushain97 commented:
> I do not know what GCI is, can you explain?

https://codein.withgoogle.com/ We have a lot of folks coming around this time related to GCI.

> I'm now starting to contribute with open source, i'm still learning how this world works.

No worries!

> I changed the routes in the Makefile, can i run the tests without giving a pull request to see i there were any errors?

`make` and `make debug`.

I think the current change in https://github.com/apertium/apertium-html-tools/pull/328/files#diff-b67911656ef5d18c4ae36cb6741b7965L113 breaks `index.debug.html`

Current master: https://590-28576409-gh.circle-artifacts.com/0/home/circleci/project/build/index.debug.html?dir=eng-epo#translation

Your branch: https://592-28576409-gh.circle-artifacts.com/0/home/circleci/project/build/index.debug.html

gustafsilva commented:
> > I do not know what GCI is, can you explain?
> 
> https://codein.withgoogle.com/ We have a lot of folks coming around this time related to GCI.
> 
> > I'm now starting to contribute with open source, i'm still learning how this world works.
> 
> No worries!
> 
> > I changed the routes in the Makefile, can i run the tests without giving a pull request to see i there were any errors?
> 
> `make` and `make debug`.
> 
> I think the current change in https://github.com/apertium/apertium-html-tools/pull/328/files#diff-b67911656ef5d18c4ae36cb6741b7965L113 breaks `index.debug.html`
> 
> Current master: https://590-28576409-gh.circle-artifacts.com/0/home/circleci/project/build/index.debug.html?dir=eng-epo#translation
> 
> Your branch: https://592-28576409-gh.circle-artifacts.com/0/home/circleci/project/build/index.debug.html

I did not know the GCI project, very cool! :smiley:
I changed the `debug-head.html` route that was missing, but i could not run the `make debug` command, but i think evething is fine now!

sushain97 commented:
> i could not run the make debug command

Why not? It would be great if you could verify this works.

gustafsilva commented:
> > i could not run the make debug command
> 
> Why not? It would be great if you could verify this works.

I tested it now by giving a `make clean` before and after `make debug` and everything worked out.

gustafsilva commented:
I did not understand @sushain97 , where would it be?
It did not find `./` references in `Makefile`.

sushain97 commented:
I'm referring the `./` prefix on file paths that is redundant due to all requirement paths being defaulted to relative to the `Makefile` path.

e.g. https://github.com/apertium/apertium-html-tools/blob/9632cb3295c3de039fa7cbffb3420e4992ca2044/Makefile#L112

can just be

    build/index.debug.html: html/index.html.in [...]
